ext/pdo/tests/bug_71885.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+PDO Common: FR #71885 (Allow escaping question mark placeholders)
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo')) die('skip');
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+if (!strncasecmp(getenv('PDOTEST_DSN'), 'pgsql', strlen('pgsql'))) die('skip not relevant for pgsql driver');
+if (!strncasecmp(getenv('PDOTEST_DSN'), 'pgsql', strlen('odbc'))) die('skip inconsistent error message with odbc');
```

Comment:
Should be odbc for the first string as well.

Reply:
@nikic Of course :(

While I have you here. Should I push to 7.4 and master or will you merge the PR?

Reply:
Feel free to merge :)

Reply:
@nikic done, thanks!